stdio/src/lib.rs

```diff
@@ -0,0 +1,44 @@
+extern crate futures;
+extern crate tokio;
+extern crate tokio_codec;
+extern crate tokio_io;
+extern crate tokio_stdin_stdout;
+
+pub extern crate jsonrpc_core;
+use jsonrpc_core::{IoHandler};
+
+use tokio::prelude::{Future, Stream};
+use tokio_codec::{FramedRead, FramedWrite, LinesCodec};
+
+fn process(io: &IoHandler, input: String) -> Box<Future<Item = String, Error = ()> + Send> {
+  Box::new(
+    io.handle_request(&input).map(move |result| match result {
+      Some(res) => res,
+      None => {
+        eprintln!("JSON RPC request produced no response: {:?}", input);
+        String::from("")
+      }
+    })
+  )
+}
+
+pub fn server(io: IoHandler) {
+  let stdin = tokio_stdin_stdout::stdin(0);
+  let stdout = tokio_stdin_stdout::stdout(0).make_sendable();
+
+  let framed_stdin = FramedRead::new(stdin, LinesCodec::new());
+  let framed_stdout = FramedWrite::new(stdout, LinesCodec::new());
+
+  let future = framed_stdin.and_then(move |line| {
```

Comment:
The whole block could look like this:
```rust
  let future = framed_stdin
	  .and_then(move |line| process(&io, line).map_err(|_| unreachable!()))
	  .forward(framed_stdout)
	  .map(|_| ())
	  .map_err(|e| panic!("{:?}", e));

  tokio::run(future)
```

Reply:
@tomusdrw Thanks for the comments! I included them and amended the commit. Can you please take another look?